src/components/Accounts/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
```

Comment:
@tomjeatt 
I'd say we can move this into `containers` as it manages internal states.

Reply:
thanks @crypto-engineer - we also have the Chains component which works in the same way, so I'll move them both over in a separate PR as part of the XCM work.